frontend/src/v5/store/containers/containers.types.ts

```diff
@@ -41,3 +43,18 @@ export interface IContainer {
 	isFavourite: boolean;
 	role: string;
 }
+
+export interface FavouritePayload {
+	teamspace: string;
+	projectId: string;
+	containerId: IContainer['_id'];
+}
+
+export interface IContainersActionCreators {
+	setFilterQuery: (query: string) => ExtendedAction<{query: string}, 'setFilterQuery'>
+	addFavourite: (teamspace: string, projectId: string, containerId: string) => ExtendedAction<FavouritePayload, 'addFavourite'>
+	removeFavourite: (teamspace: string, projectId: string, containerId: string) => ExtendedAction<FavouritePayload, 'removeFavourite'>
+	toggleFavouriteSuccess: (containerId: string) => ExtendedAction<{containerId: string}, 'toggleFavouriteSuccess'>
+}
+
+export type IContainersActions = AllReturnTypes<IContainersActionCreators>;
```

Comment:
Same to this, I prefer to have the reducers like setFilterQuery() in the style of 
`export const setFilterQuery = (state = INITIAL_STATE, { query }: FilterQueryAction)`